### ISSUE TITLE
Plugin still being developed?

### ISSUE THREAD

**Issue by RossAscends** [SheepGoMeh/VisibilityPlugin#77]
After updating to Dawntrail, and bringing Dalamud and all plugins up to date, Visibility's voidlist no longer works to hide players. 

Usually I manage Whitelist/voidelist via Player Tracker plugin, but this is also the case when using the visibility settings UI or slash commands directly.

In in latest Player Tracker update the UI has a note stating that VIsibility integration will be removed in their next update "because Visibility is depreciated."

Is this to say the plugin is no longer being developed, and is currently not effective? If this is the case, do you know of another plugin which has the same kind of functionality?

**Comment by kalilistic** [comment]
Clarified in chat, but in the PlayerTrack notes I said _Voidlist_ is deprecated, not the whole Visibility plugin. The vanilla blacklist was improved with dawntrail and can be used instead.

**Comment by PrincessRTFM** [comment]
Voidlist has been deprecated and disabled, as kalilistic said. The rest of the plugin's functionality is and will be maintained.

**Comment by pictochatting** [comment]
considering vanilla blacklist has limitation of 200 players, i believe there is still value in visibility voidlist. 

additionally, it's jarring to see everyone i've voidlisted show up again - having to go through the not-so-easy vanilla way to blacklist someone (can't right click blacklist, need to player search, etc. there's just way more friction here than /voidtarget)

for many people, they use voidlist as a tool against harassment, so safe to say it's removal unfortunately caused some churn & harm. especially without it being communicated. (e.g. the voidlist is still there in the plugin, commands are still listed - i'm sure i'm not the only one who's confused)

i understand vanilla blacklist is "better", but can you reconsider the decision to deprecate voidlist? 

thank you for the work on visibility plugin!

**Comment by PrincessRTFM** [comment]
As one of Sheep's co-developers, I can say with quite a bit of certainty that voidlist will _not_ be returning. The feature was complex and difficult to do well, because _every single frame_, every user in the area needed to be checked by name (and string comparisons are, at this level, surprisingly expensive) against _every_ name on the list. I know for a fact that Sheep did a lot of work optimising to avoid absolutely tanking framerates, because they talked to me about it while working on it. On top of that, a lot of users consistently complained, or found problems with it that required significant effort to fix, or just added so many users to their lists that their FPS died and then blamed Sheep for it.

**Comment by pictochatting** [comment]
thanks, and I respect your decision to do what y'all want with the plugin.

**Comment by SheepGoMeh** [comment]
There was a slight misunderstanding with my intent;
I will keep VoidList functional at the bare minimum level for the people who actually rely upon it for important things like stalkers and so on.
However, I will not be updating the functionality on it going further as the game's blacklist should do a better job when it comes to things like alt characters.
I strongly advise migrating your VoidList to the vanilla `Mute List` and `Blacklist` features.

tl:dr;
VoidList will function, but will not be updated in the future.
